src/web/components/Approvals/ApprovalItem.scss

```diff
@@ -0,0 +1,31 @@
+.approval-item {
+  line-height: 2rem;
```

Comment:
Probably not an ideal way to get the rows the same height (at at least look the same height) as the other tables. A lot of the scss has been copied from elsewhere then just changed to look how I think it should.

Reply:
Changed to just be 2, instead of 2rem.